components/datetime/src/provider/time_zones.rs

```diff
@@ -42,13 +42,12 @@ pub(crate) mod tz {
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[yoke(prove_covariance_manually)]
 pub struct TimeZoneEssentialsV1<'data> {
-    /// The hour format for displaying offsets.
-    #[cfg_attr(feature = "serde", serde(borrow))]
-    #[cfg_attr(
-        feature = "serde",
-        serde(deserialize_with = "icu_provider::serde_borrow_de_utils::tuple_of_cow")
-    )]
-    pub hour_format: (Cow<'data, str>, Cow<'data, str>),
+    /// The sign for positive offsets
+    pub offset_positive_sign: char,
+    /// The sign for negative offsets
+    pub offset_negative_sign: char,
+    /// The separator sign
+    pub offset_separator: char,
```

Comment:
I gave "permission" to flatten `offset_separator` to a char. The other two seem more likely to be non-chars (e.g. Arabic sometimes has a two-code-point plus sign); a `SinglePlaceholderPattern` would be more appropriate, though heavier on the stack size.

But, if it works with today's data, I'm not super concerned. Our stability policy gives us a pathway to fix this in the future if we need to.